src/RoslynPad.Hosting/ExecutionHost.cs

```diff
@@ -156,14 +159,18 @@ private static DelegatingTextWriter CreateConsoleWriter()
 
         public ExecutionHost(string hostPath, string initialWorkingDirectory,
```

Comment:
`CSharpCompilationOptions` is too low-level to use here IMO. It's also not serializable.